tests/testthat/_snaps/btw.md

```diff
@@ -3,5 +3,5 @@
     Code
       print(btw(mtcars))
```

Comment:
Might want to also add `print(btw(get_data_frame(mtcars, format = "glimpse"))` to these tests so we hit all `format`s